src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/InternalCaseCreationContactDetailsAppender.java

```diff
@@ -53,6 +55,26 @@ public PreSubmitCallbackResponse<AsylumCase> handle(
             if (internalAppellantEmail.isPresent()) {
                 asylumCase.write(EMAIL, internalAppellantEmail);
             }
+
+            Optional<String> appealSubmissionInternalDate = asylumCase.read(APPEAL_SUBMISSION_INTERNAL_DATE, String.class);
+            if (appealSubmissionInternalDate.isPresent()) {
+                asylumCase.write(APPEAL_SUBMISSION_INTERNAL_DATE, appealSubmissionInternalDate);
+            }
+
+            Optional<String> decisionLetterReceivedDate = asylumCase.read(DECISION_LETTER_RECEIVED_DATE, String.class);
+            if (decisionLetterReceivedDate.isPresent()) {
+                asylumCase.write(APPEAL_SUBMISSION_INTERNAL_DATE, decisionLetterReceivedDate);
+            }
+
+            Optional<String> homeOfficeDecisionDate = asylumCase.read(HOME_OFFICE_DECISION_DATE, String.class);
+            if (homeOfficeDecisionDate.isPresent()) {
+                asylumCase.write(APPEAL_SUBMISSION_INTERNAL_DATE, homeOfficeDecisionDate);
+            }
```

Comment:
APPEAL_SUBMISSION_INTERNAL_DATE = Appeal created date. Why is it being copied from three different case fields?